apps/earn-protocol/helpers/vault-decorators/chart-historical-data.ts

```
@@ -0,0 +1,293 @@
+              ...chartsDataRaw['3y'][timestamp],
+              [arkUniqueName]: Number(weeklyInterestRate.averageRate),
+            }
+          }
+        }
+      }
+      const chartDataPoints = Object.keys(chartsDataRaw).reduce<ChartsDataTimeframes>(
+        (acc, key) => {
+          const data = Object.values(chartsDataRaw[key as keyof typeof chartsDataRaw])
+
+          return {
+            ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)